### DISCUSSION TITLE
MAXIMUM PRESSURE AND VELOCITY ERRORS

### DISCUSSION THREAD

**Discussion by MarioCruzM** [firemodels/fds#13420] [General]
So,  I'm trying to establish a reaction for heavy fuel oil (HFO) pyrolisis. I'm working on Pyrosim and so far I have all the inputs for the reaction, Nonetheless, when I try to run the simple simulation, I get the error **forrtl: severe (157): Program Exception - access violation**. In the model I'm uploading I do have sprinklers and I know this error can come from a bad input on the sprinklers, but I've tried running the simulation without the sprinklers and I get the same error. I even copied the formula only into an old model that I know works without any problem but I got the same error, so I'm pretty confident that something is going on with the reaction inputs, so far I haven't been able to find this error.

FYI: for this simulation, I'm assuming a formula of C20H2.30 this formula came from an elemental analysis, which established the proportions of carbon to be 88.98 and H 10.25, I then simplified this formula to C20 because that's the average length of hydrocarbons on HFO. I've also tried simplifying the formula to C1H0.1152 (I read that using a heavy molecular weight can cause problems) but I did get the same error.

I know it's a very specific case so I really appreciate anyone's help.
[TEST_HFO.zip](https://github.com/user-attachments/files/16950441/TEST_HFO.zip)

**Comment by mcgratta** [comment]
Next time, submit as an issue. I will run the case.

**Reply by MarioCruzM** [reply]
Noted; thank you for the observation.

**Comment by mcgratta** [comment]
Also, you've given me the ini file. I need the .fds file.

**Reply by MarioCruzM** [reply]
Sorry, my bad,
[TEST_HFO.zip](https://github.com/user-attachments/files/16951424/TEST_HFO.zip)

**Comment by mcgratta** [comment]
This is now an [issue](https://github.com/firemodels/fds/issues/13421).
